src/commands/download_drive.ts

```diff
@@ -0,0 +1,48 @@
+import { EID } from 'ardrive-core-js';
+import { cliArDriveAnonymousFactory, cliArDriveFactory } from '../index';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import { DrivePrivacyParameters, DriveIdParameter, LocalPathParameter } from '../parameter_declarations';
+import { getOutputFolderPathAndName } from '../utils';
+import { join as joinPath } from 'path';
+
+new CLICommand({
+	name: 'download-drive',
+	parameters: [DriveIdParameter, { name: LocalPathParameter, description: 'TODO' }, ...DrivePrivacyParameters],
+	action: new CLIAction(async (options) => {
+		const parameters = new ParametersHelper(options);
+		const driveId = parameters.getRequiredParameterValue(DriveIdParameter, EID);
+		const maxDepth = await parameters.getMaxDepth(Number.MAX_SAFE_INTEGER);
+		const destOutputPath = parameters.getParameterValue(LocalPathParameter) || '.';
```

Comment:
Won't this have the wrong description text?

Reply:
If it was regarding the parameter description, then: 4f6700e5d4eb8239d30f6f0897ebcb34a5c3182e